pkg/storage/storage_fs.go

```diff
@@ -1641,3 +1643,58 @@ func DirExists(d string) bool {
 
 	return true
 }
+
+func GetValidGCInterval(providedGCInterval time.Duration, log *zlog.Logger) time.Duration {
+	if providedGCInterval < DefaultGCInterval {
+		log.Warn().Msg("GC interval set to too-short interval < 2h, changing GC duration to 2 hours and continuing.") // nolint: lll
+
+		return DefaultGCInterval
+	}
+
+	return providedGCInterval
+}
+
+func (is *ImageStoreFS) gcAllRepos() error {
```

Comment:
make this a function like ifOlderThan() and give it the imageStore as an argument

Reply:
otherwise LGTM